LRRServer/lawreadingrobot/crud.py

```diff
@@ -0,0 +1,34 @@
+
```

Comment:
Please add a short file comment https://google.github.io/styleguide/pyguide.html#382-modules
> Files should start with a docstring describing the contents and usage of the module.

Reply:
Seconded. If Chris gets abducted by aliens, we gotta make sure new people coming into it have a decent context in case we can't reach him 👽.